db/migrate/20170828133538_create_timetracks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTimetracks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :timetracks do |t|
+      t.string  :name
+      t.text    :description
+      t.decimal :work_time, precision: 5, scale: 2
```

Comment:
Besser auch auf `NOT NULL` setzen: `t.decimal :work_time, precision: 5, scale: 2, null: false` (Default ist bereits `0`).